src/js/msp/MSPCodes.js

```diff
@@ -8,8 +8,8 @@ const MSPCodes = {
     MSP_BOARD_INFO:                 4,
     MSP_BUILD_INFO:                 5,
 
-    MSP_NAME:                       10,
-    MSP_SET_NAME:                   11,
+    MSP_NAME:                       10, // DEPRECATED IN MSP 1.45
+    MSP_SET_NAME:                   11, // DEPRECATED IN MSP 1.45
```

Comment:
Dunno if it will work straight away, but we could use jsdoc to add some intelissense about these.
```suggestion
    /**
     * @deprecated in MSP >= 1.45
     */
    MSP_NAME:                       10,
    /**
     * @deprecated in MSP >= 1.45
     */
    MSP_SET_NAME:                   11,
```